editor/plugins/game_view_plugin.cpp

```diff
@@ -96,6 +96,14 @@ void GameViewDebugger::_session_started(Ref<EditorDebuggerSession> p_session) {
 	mode.append(select_mode);
 	p_session->send_message("scene:runtime_node_select_set_mode", mode);
 
+	Array shortcuts;
+	Dictionary shortcut_settings;
+	shortcut_settings["editor/suspend_resume_embedded_project"] = DebuggerMarshalls::serialize_key_shortcut(ED_GET_SHORTCUT("editor/suspend_resume_embedded_project"));
+	shortcut_settings["editor/next_frame_embedded_project"] = DebuggerMarshalls::serialize_key_shortcut(ED_GET_SHORTCUT("editor/next_frame_embedded_project"));
+
+	shortcuts.append(shortcut_settings);
+	p_session->send_message("scene:setup_embedded_shortcuts", shortcuts);
```

Comment:
```suggestion
	p_session->send_message("scene:setup_embedded_shortcuts", { shortcut_settings });
```
`shortcuts` is no longer needed.

Reply:
Not really sure what do you mean by that?
The shortcuts Array is there to encapsulate shortcut_settings to `send_message` as that only accepts an Array&.
Following the example of what "scene:runtime_node_select_setup" does just above.

Reply:
Well check my suggestion.
You don't need to create the Array beforehand, you can send the value directly (`{}` will automatically construct the array).

Reply:
Oh, thanks for that. 
I swear I had tried your suggestion before and was getting an error saying it could not convert the value type.
But after updating master now it works.